stable/pachyderm/templates/dash_deployment.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.dash.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: dash
+  labels:
+    app: {{ template "dash.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    suite: {{ template "fullname" . }}
```

Comment:
Should we add `annotations` here?

Reply:
Neither the `pachd` or `etcd` deployments have annotations there so I wouldn't add it in this PR.